src/EnergyPlus/DataPlantPipingSystems.cc

```diff
@@ -113,6 +113,18 @@ namespace DataPlantPipingSystems {
 	Array1D< PipeCircuitInfo > PipingSystemCircuits;
 	Array1D< PipeSegmentInfo > PipingSystemSegments;
 
+	//*********************************************************************************************!
+
+	//*********************************************************************************************!
+
+	void
+	clear_state()
+	{
+		Array1D< FullDomainStructureInfo > PipingSystemDomains;
+		Array1D< PipeCircuitInfo > PipingSystemCircuits;
+		Array1D< PipeSegmentInfo > PipingSystemSegments;
```

Comment:
And just verify that there aren't any other lingering "module-level" variables to be cleaned up in preparation for subsequent unit tests.